dev/run-dev-server.sh

```diff
@@ -1,32 +1,28 @@
 #!/bin/bash
 set -e
 
-log_file="outputs/mlflow-server.log"
-
 function wait_server_ready {
   for backoff in 0 1 2 4 8; do
-    echo "Waiting for server to be ready..."
+    echo "Waiting for tracking server to be ready..."
     sleep $backoff
     if curl --fail --silent --show-error --output /dev/null $1; then
       echo "Server is ready"
       return 0
     fi
   done
-  cat $log_file
-  echo -e "\nFailed to launch mlflow server"
+  echo -e "\nFailed to launch tracking server"
   return 1
 }
 
 mkdir -p outputs
-echo 'Running mlflow server in the background'
-echo "Logging to $log_file"
+echo 'Running tracking server in the background'
 if [ -z "$MLFLOW_TRACKING_URI" ]; then
   backend_store_uri=""
   default_artifact_root=""
 else
   backend_store_uri="--backend-store-uri $MLFLOW_TRACKING_URI"
   default_artifact_root="--default-artifact-root mlruns"
 fi
-mlflow server $backend_store_uri $default_artifact_root --gunicorn-opts="--log-level debug" > $log_file 2>&1 &
+mlflow server $backend_store_uri $default_artifact_root --gunicorn-opts="--log-level debug --reload" &
```

Comment:
Q: How to kill the server easily ?

Reply:
Ctrl + C

Reply:
<img width="585" alt="image" src="https://user-images.githubusercontent.com/17039389/180712045-e10ca27e-6f27-41a5-8f1c-50c43e8c0ee9.png">

Reply:
I think we can do:

```
spawnedChildProcess1 &
spawnedChildProcess2 &
spawnedChildProcess3 &

wait
```
in the script, so when the script process exit, all children process will also be killed.

Reply:
You added `&` after the `mlflow server` command, I guess ctrl+c does not work to kill it

Reply:
gunicorn automatically kills the child processes when the script process exits.

Reply:
@WeichenXu123 You can clone this PR and test it.

Reply:
@WeichenXu123 Did you test it?

Reply:
Yes. Your PR works. Interesting. gunicorn should catch parent process signal.